tensorrt_llm/_torch/auto_deploy/transform/library/mxfp4_moe.py

```
@@ -282,11 +559,44 @@ def _apply(
+            source_names = {
+                "gate_up_weight": gu_w_name,
+                "gate_up_bias": gu_b_name,
+                "down_weight": dn_w_name,
+                "down_bias": dn_b_name,
+            }
+            layer = None
+            if checkpoint_layout is not None:
+                layer = _layout_layer_from_names(checkpoint_layout, source_names)
+                if layer is None:
+                    continue
+            num_experts, hidden_size, intermediate_size = _get_mxfp4_expert_dims(
```

Ruff (0.15.15)

[warning] Unpacked variable `num_experts` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)